spec-main/api-net-spec.ts

```diff
@@ -592,8 +592,9 @@ describe('net module', () => {
           url: `${serverUrl}`,
           name: 'test',
           value: '11111',
-          expirationDate: 0
-        }).then(() => { // resolved
+          expirationDate: new Date().getTime() + 30000
```

Comment:
Leaving out the expiration date should be fine since https://github.com/electron/electron/pull/21454